### ISSUE TITLE
No Output of Audio

### ISSUE THREAD

**Issue by nighthawk9** [okonek/tidal-cli-client#27]
Distro: KDE Neon (Ubuntu 16.04 LTS)
Terminal Emulator: (Konsole, xTerm)
Node Version: 9.11.1
tidal-cli Version: 1.4.6

I am receiving no audio output from the application. I have check to make sure I have all of the audio dependencies such as MPV and node-mpv specifically. I have confirmed that MPV works.

Any idea of what would be causing this issue? If more input is required or anything I can help with, just let me know. 

Thank you,

Michael

**Comment by okonek** [comment]
When you select a song, does it display its name in the bottom bar?

**Comment by nighthawk9** [comment]
Yes, the song is displayed in the bottom bar. If I add in more songs to the queue and then play them, the title of the song changes at 0.5 s intervals until the end of the queue.

I have tried playing songs both from albums and through searching tracks.

**Comment by okonek** [comment]
Have you got a Lossless quality?

**Comment by nighthawk9** [comment]
Yes, I have selected lossless quality and hold a subscription to listen to the quality. I also tried without lossless and it produces the same result of no audio.

**Comment by okonek** [comment]
Could you please try running this script, to tell if node-mp works on your computer? After uncompressing, you should do `npm install` and then run `node mpv-test.js ch.wav`. ch.wav is a Chopin etude. Please replay if you hear anything.
[mpv-test.tar.gz](https://github.com/okonek/tidal-cli-client/files/1911918/mpv-test.tar.gz)

**Comment by okonek** [comment]
I suggest you to uninstall the tidal-cli-client and mpv from your system. Then firstly install mpv and then tidal-cli-client. Tell me if it worked or not.
